wsproto/extensions.py

```diff
@@ -246,3 +246,8 @@ def __repr__(self):
         descr = '; '.join(descr)
 
         return '<%s %s>' % (self.__class__.__name__, descr)
+
+
+SUPPORTED_EXTENSIONS = {
+    PerMessageDeflate.name: PerMessageDeflate
+}
```

Comment:
Yes, for future-proofing.
Yes, for being public API.

Although at the moment there is only one extension in wsproto, and probably only one extension at all for WebSockets (any other known RFC's?), it would be nice to have something to loop through and iterate all supported extensions, and their base classes.
Otherwise users need to know & hard-code all extensions, with such a dict, I could simply loop through them to activate everything.
In @mitmproxy we can also make use of this to quickly check if a handshake contains any unsupported extensions by comparing it against this dict's keys.

Again, a single extension is no real use-case. But why not?